packages/analytics-nextjs/src/context.tsx

```diff
@@ -137,6 +139,7 @@ export function AnalyticsContextProvider({
                             domain: document.location.host,
                             ...cookie,
                         },
+                        user,
```

Comment:
I think it should be good like this? 🤔 Even if `user` is an empty object, the library should fallback to the defaults.

Reply:
I'd leave the default `undefined` value. The library might not be that smart :)

Reply:
Changed it 👍